rewriter/Prop.cc

```diff
@@ -48,6 +48,7 @@ struct PropInfo {
     core::NameRef computedByMethodName = core::NameRef::noName();
     core::LocOffsets computedByMethodNameLoc;
     unique_ptr<ast::Expression> foreign;
+    unique_ptr<ast::Expression> ifunset;
```

Comment:
What does the value of `ifunset` get used for? I only see it tested against `nullptr` in this file.

Reply:
It's kind of like `default:`. We will need it eventually.